pkg/agent/openflow/pipeline.go

```diff
@@ -346,13 +348,12 @@ func (c *client) podClassifierFlow(podOFPort uint32, category cookie.Category) b
 // hostBridgeUplinkFlows generates the flows that forward traffic between bridge local port and uplink port to support
 // host communicate with outside. These flows are only needed on windows platform.
 func (c *client) hostBridgeUplinkFlows(uplinkPort uint32, bridgeLocalPort uint32, category cookie.Category) (flows []binding.Flow) {
-	classifierTable := c.pipeline[ClassifierTable]
 	flows = []binding.Flow{
-		classifierTable.BuildFlow(priorityLow).MatchInPort(uplinkPort).
+		c.pipeline[uplinkTable].BuildFlow(priorityMiss).MatchInPort(uplinkPort).
```

Comment:
Do you think this flow entry could move to the uplinkTable, and move the one in line 964 here?

Reply:
Done.